lib/summarize.js

```diff
@@ -67,7 +72,7 @@ module.exports = function summarize(summaryPath) {
       if (isUglifyable) {
         return uglify(concatenatedContent);
       } else {
-        return concatenatedContent;
+        return normalizeLineEndings(concatenatedContent);
```

Comment:
Could we apply `normalizeLineEndings()` at line 43, in which case I believe we wouldn't have to do it here and at L87 anymore, right?